index.js

```diff
@@ -9,15 +9,15 @@
 require('dotenv').config();
 const http = require('http');
 const express = require('express');
-const env = require('./config.js').env;
+const { env, port } = require('./config.js');
 const config = require('./config.js')[env];
 const packageJSON = require('./package.json');
 const bdk = require('@salesforce/refocus-bdk')(config);
 const salesforce = require('./salesforce');
 const botName = packageJSON.name;
 
 const app = express();
-const { PORT, enableDropzone } = config;
+const { enableDropzone } = config;
```

Comment:
then here enableDrop = env; // maybe renaming envConfig? something related to configuration per env...